### DISCUSSION TITLE
Feedback - IBM

### DISCUSSION THREAD

**Discussion by Retr0Spec-dev** [WillTheDeveloper/StudyPortal#15] [General]
works well.

i like the improvement of the gui, and the other tabs as it looks clean as well as it seems to be easy to navigate

**Comment by WillTheDeveloper** [comment]
Thanks for your feedback :) Do you have any suggestions or anything that you dislike? @Retr0Spec-dev

**Reply by Retr0Spec-dev** [reply]
adding a more diverse colour range might be good or have a colour customisation setting for each user

**Reply by WillTheDeveloper** [reply]
Okay. Do you believe that would add more value to the platform in your opinion?

**Reply by Retr0Spec-dev** [reply]
mayhaps

**Reply by WillTheDeveloper** [reply]
Okay, thanks!
